### PR TITLE
Ability to get parsed json-rpc request

### PR DESCRIPTION
**Pull request by smagafurov**

### CODE REVIEW

File: jsonrpc/base.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ class JSONRPCBaseResponse(JSONSerializable):
 
     """ Base class for JSON-RPC 1.0 and JSON-RPC 2.0 responses."""
 
+    request = None  # type: JSONRPCBaseRequest
```
Comment (pavlov99) [inline jsonrpc/base.py]: Why does "request" attribute belong to class? Should it be implemented on the instance level in __init__ method?

File: jsonrpc/manager.py [hunk 2]
```diff
@@ -96,8 +98,10 @@ def _get_responses(cls, requests, dispatcher):
         """
         for request in requests:
             def response(**kwargs):
-                return cls.RESPONSE_CLASS_MAP[request.JSONRPC_VERSION](
+                resp = cls.RESPONSE_CLASS_MAP[request.JSONRPC_VERSION](
```
Comment (pavlov99) [inline jsonrpc/manager.py]: May you call `resp` as `response` please? This is absolutely minor issue but it ensures the consistency.

File: jsonrpc/base.py (hunk 1 above)
Comment (smagafurov) [reply jsonrpc/base.py]: fixed

File: jsonrpc/manager.py (hunk 2 above)
Comment (smagafurov) [reply jsonrpc/manager.py]: fixed

**Comment by smagafurov** [comment]
@pavlov99 Travis failed with strange error "An error occurred while generating the build script."

**Comment by pavlov99** [comment]
@smagafurov, It was merged with rebase commit. Because this pull request was not rebased properly it introduced several issuer with merge to master. Currently they are resolved.

**Comment by pavlov99** [comment]
@smagafurov new version released: https://pypi.python.org/pypi/json-rpc/1.10.7
Thanks for your contribution.

**Pull request closed without merging**